lib/utils/getServerlessConfigFile.js

```diff
@@ -6,8 +6,14 @@ const path = require('path');
 const fileExists = require('./fs/fileExists');
 const readFile = require('./fs/readFile');
 
-const getServerlessConfigFilePath = (srvcPath) => {
+const getServerlessConfigFilePath = (configFileName, srvcPath) => {
```

Comment:
I see why we're implementing it that way here, but isn't there any way to move the logic to make the distinction between `options.config` and `config.servicePath` into another layer?

This change means that you always need to have the `options` and the `servicePath` available to pass it down the function. Would be nice to just give it the path to the config file and this function figures out what to do.

Reply:
you mean implement process.argv parsing in this function? I .. don't like that :/

Reply:
No, I mean consolidating that into just one parameter, rather than having 2 parameters (right now we pass in parts of the `options` and the `servicePath`).

Reply:
Done. It just accepts `this.serverless` from things using it.